src/Symfony/Component/Validator/Resources/translations/validators.ro.xlf

```diff
@@ -398,6 +398,30 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>Valoarea netmask-ului trebuie sa fie intre {{ min }} si {{ max }}.</target>
             </trans-unit>
+            <trans-unit id="104">
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>Denumirea fișierului este prea lungă. Ea trebuie să conțină {{ filename_max_length }} caractere sau mai puține.|Denumirea fișierului este prea lungă. Ea trebuie să conțină {{ filename_max_length }} caractere sau mai puține.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>The password strength is too low. Please use a stronger password.</source>
+                <target>Complexitatea parolei este prea mică. Vă rugăm să folosiți o parolă mai puternică.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>This value contains characters that are not allowed by the current restriction-level.</source>
+                <target>Această valoare conține caractere care nu sunt premise de nivelul de restricționare curent.</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Using invisible characters is not allowed.</source>
+                <target>Folosirea caracterelor invizibile nu este permisă.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Mixing numbers from different scripts is not allowed.</source>
+                <target>Combinarea numerelor din diferite stript-uri nu este permisă.</target>
```

Comment:
typo: `script-uri`

Reply:
This one was the trickiest to translate for me. I checked other translation files and I saw many use the `script` word as the root. The problem is if we use the word `scripturi` and it is read with an accent on `u` instead of `i`, the meaning will become `religious writing`, which I tried to avoid. Maybe it makes more sense to translate the word as it was done for Russian? So it will become `scenarii` ?